api/tutorial.js

```diff
@@ -1,7 +1,7 @@
 const tutorialDB = require('../db/tutorial-queries')
 const authCheck = require('../utils/authentication-check')
 
-function createTutorial (req, res, next) {
+function createTutorial (req, res) {
```

Comment:
Is there a reason to change this?

Reply:
It's redundant and other functions in the file do not have it.

Reply:
My bad. nvm